### ISSUE TITLE
textlint fails to resolve allowlistConfigPaths when the fix flag is specified

### ISSUE THREAD

**Issue by yujiorama** [textlint/textlint-filter-rule-whitelist#10]
## environment

* OS Name: Microsoft Windows 10 Pro
* OS Version: 10.0.19042 N/A Build 19042
* Node: v14.15.4
* Npm: 6.14.10
* textlint: 11.8.1
* textlint-filter-rule-allowlist: 2.0.1
* textlint-rule-prh: 5.3.0

## problems

textlint fails to resolve allowlistConfigPaths when the fix flag is specified.
(textlint success to resolve prh's rulePaths when the fix flag is specified.)

[example-repository](https://github.com/yujiorama/fails-to-resolve-allowlistConfigPaths)

```bash
npx textlint --config lint/.textlintrc doc/a.md
× Error
Error while loading filter rule 'allowlist': This allow file is not allow word list: ./allowlist/allow.json
at C:\Users\y_okazawa\tmp\allowlist\doc\a.md

× Stack trace
Error: Error while loading filter rule 'allowlist': This allow file is not allow word list: ./allowlist/allow.json
at C:\Users\y_okazawa\tmp\allowlist\doc\a.md
    at C:\Users\y_okazawa\tmp\allowlist\node_modules\textlint-filter-rule-allowlist\lib\textlint-filter-rule-allowlist.js:28:13
    at Array.forEach (<anonymous>)
    at getAllowWordsFromFiles (C:\Users\y_okazawa\tmp\allowlist\node_modules\textlint-filter-rule-allowlist\lib\textlint-filter-rule-allowlist.js:19:9)
    at module.exports (C:\Users\y_okazawa\tmp\allowlist\node_modules\textlint-filter-rule-allowlist\lib\textlint-filter-rule-allowlist.js:60:61)
    at TextLintCoreTask.tryToGetFilterRuleObject (C:\Users\y_okazawa\tmp\allowlist\node_modules\@textlint\kernel\lib\task\textlint-core-task.js:191:20)
    at TextLintCoreTask.tryToAddListenRule (C:\Users\y_okazawa\tmp\allowlist\node_modules\@textlint\kernel\lib\task\textlint-core-task.js:209:20)
    at C:\Users\y_okazawa\tmp\allowlist\node_modules\@textlint\kernel\lib\task\fixer-task.js:67:19
    at Array.forEach (<anonymous>)
    at TextLintCoreTask._setupRules (C:\Users\y_okazawa\tmp\allowlist\node_modules\@textlint\kernel\lib\task\fixer-task.js:60:48)
    at new TextLintCoreTask (C:\Users\y_okazawa\tmp\allowlist\node_modules\@textlint\kernel\lib\task\fixer-task.js:36:15)
```

## additional note

textlint success without the fix flag.

```bash
npx textlint --config lint/.textlintrc doc/a.md

C:\Users\y_okazawa\tmp\fails-to-resolve-allowlistConfigPaths\doc\a.md
  7:1  ✓ error  テスティング => テスト  prh

✖ 1 problem (1 error, 0 warnings)
✓ 1 fixable problem.
Try to run: $ textlint --fix [file]
```

**Comment by azu** [comment]
Thanks for the report.
It look like a bug of textlint.
I've created an issue.
https://github.com/textlint/textlint/issues/741

**Comment by azu** [comment]
I've fixed it in [textlint@11.8.2](https://github.com/textlint/textlint/releases/tag/textlint%4011.8.2)

Please update textlint via `npm update textlint`.
